www/js/controller.js

```diff
@@ -596,24 +596,19 @@ define([
 				var d = registry.byId(this.controlMap.description);
 				var v = registry.byId(this.controlMap.variable);
 				array.forEach(this._model.authored.getDescriptions(), lang.hitch(this, function(desc){
-					var exists =  model.getNodeIDFor(desc.value);
-
-					if( d.getOptions !== undefined && d.getOptions(desc)) {
-						d.getOptions(desc).disabled=exists;
-						if(desc.value == nodeName){
-							d.getOptions(desc).disabled=false;
+					if(this._model.doesStudentNodeExist(desc.value)){
```

Comment:
this code will be called for author mode as well. what happens then?

Reply:
I am saying as this is in controller.js

Reply:
This part is never executed in author mode as model.getNodeIDFor is not defined for author